docs/source/contents/4_draft_mapping.rst

```diff
@@ -1662,6 +1662,407 @@ opaque:hostItem
++-----------------------------------+----------------+-------------------+-------------------------------------------------------------------------+
+| skos:note                         | Literal        |                   | Use to note <mods:shelfLocator> strings.                                |
++-----------------------------------+----------------+-------------------+-------------------------------------------------------------------------+
+| dbo:collection                    | Literal        |                   | Use to note <mods:physicalLocation[@displayLabel=Collection]> strings.  |
++-----------------------------------+----------------+-------------------+-------------------------------------------------------------------------+
+
+physicalLocation as URI
+-----------------------
+
+Use Case
+^^^^^^^^
+Many of records have valueURI attributes set for physicalLocation. This is inconsistent, even in our own collections.'
+
+Justification
+^^^^^^^^^^^^^
+When available, we will opt to use valueURI values as the URI value for relators:rps, to better qualify objects of relators:rps.
+
+Xpath
+^^^^^
+
+`mods:location/mods:physicalLocation[@valueURI=https://example.org/authority/1]`
+
+Decision
+^^^^^^^^
+The valueURI attribute `mods:location/mods:physicalLocation` is set as object.
+
+`Example record from egypt:79 <https://digital.lib.utk.edu/collections/islandora/object/egypt:79/datastream/MODS/view>`_
+
+.. code-block:: xml
+
+    <location>
+        <physicalLocation valueURI="http://id.loc.gov/authorities/names/no2017033007">Frank H. McClung Museum of Natural History and Culture</physicalLocation>
+    </location>
+
+.. code-block:: turtle
+
+    @prefix relators: <http://id.loc.gov/vocabulary/relators> .
+
+    <https://example.org/objects/1>
+        relators:rps <http://id.loc.gov/authorities/names/no2017033007> .
+
+physicalLocation as string (UTK)
+--------------------------------
+
+Use Case
+^^^^^^^^
+In many of our own collections, we use strings to describe physicalLocation. We are inconsistent in the structuring of these strings, leading to many different variations for both our Libraries and Special Collections, some of these include misspellings or just different formatting:
+
+- The University of Tennessee Libraries, Knoxville
+- University of Tennesse Knoxville. Libraries
+- University of Tennessee Knoxville. Libraries
+
+Justification
+^^^^^^^^^^^^^
+To create better consistency and cleanliness going forward, we will isolate all instances of these strings and transcribe them to appropriate URIs for UT Libraries and Special collections.
+
+Xpath
+^^^^^
+
+`mods:location/mods:physicalLocation`
+
+Decision
+^^^^^^^^
+Even when mods only has a string present, we will work to map The University of Tennessee Libraries, Knoxville and University of Tennessee, Knoxville. Special Collections to relative URIs.
```

Comment:
I'm not sure if "work to map" means deal with during migration or remediate prior to  or after migration. I can add a ticket to the MODS to RDF epic to address these inconsistencies if we want to deal with them through remediation.